service/src/main/java/bio/terra/cbas/dependencies/wds/WdsService.java

```diff
@@ -57,6 +83,131 @@ public RecordResponse updateRecord(
         });
   }
 
+  public Capabilities getCapabilities(BearerToken userToken) throws WdsServiceException {
+    return executionWithRetryTemplate(
+        listenerResetRetryTemplate, () -> wdsClient.capabilitiesApi(userToken).capabilities());
+  }
+
+  public boolean useSearchByIdsFilter(BearerToken userToken) {
+    try {
+      Capabilities wdsAppInstanceCapabilities = getCapabilities(userToken);
+      Map<String, Object> additionalProperties =
+          wdsAppInstanceCapabilities.getAdditionalProperties();
+
+      return additionalProperties != null && additionalProperties.containsKey("search.filter.ids");
```

Comment:
Ideally you'd also check here that the value of `search.filter.ids` is true, instead of just checking that the key exists.

Reply:
Added this check in [this commit](https://github.com/DataBiosphere/cbas/pull/289/commits/09962fe41d91cfbcd1bc5a66467f1acb251d6ab6)